added exam report (#138)
yayoi-chi/yaya-world: content/en/news/2025/initial-exam-report.org

#+TITLE: Initial Exam Report
#+DATE: 2025-08-10T09:39:44-07:00
#+DRAFT: false
#+DESCRIPTION:
#+TAGS[]:
#+KEYWORDS[]:
#+SLUG:

> #+SUMMARY: Hello every kitten! It has been four weeks since I started exams and I think its time  for an update. I know I've been giving little mini-updates on stream, but thought maybe something a bit more formal would be nice. Also, some changes are coming!

Reviewer: extra space
```suggestion
#+SUMMARY: Hello every kitten! It has been four weeks since I started exams and I think its time for an update. I know I've been giving little mini-updates on stream, but thought maybe something a bit more formal would be nice. Also, some changes are coming!
```

> Hello every kitten! It has been four weeks since I started exams and I think its time  for an update. I know I've been giving little mini-updates on stream, but thought maybe something a bit more formal would be nice. Also, some changes are coming!

Reviewer: ```suggestion
Hello every kitten! It has been four weeks since I started exams and I think its time for an update. I know I've been giving little mini-updates on stream, but thought maybe something a bit more formal would be nice. Also, some changes are coming!
```

So I got my questions a day and a half later and had to scramble to try and get more writing in so I could be on time. Fortunately, I did get a day extension on my due date so I wasn't too worried about it. I also managed to keep a good pace though the answer to this first question is a little shorter than I wanted. I'm not sure what more I could add though, so I moved on to the next question.

* Weeks 2 & 3
Monday I finished proofreading the first question so I didn't start the second until Tuesday. This question was a long one and it was very easy to write a lot. There was so much to cover that it was almost overwhelming. The first week some of my "writing" sessions were actually library sessions, but this time I spent a lot more time writing and less time at the library. I guess I just didn't have any trouble locating citations in references I already have. The next week was really just more of the same only I did have to spend one day at the library. By the end of the third week though I had all three questions finished and proofread.

* Week 4

> Up until this point each of my answers were in a separate document in Apple Pages. I prefer Pages because I know where everything is, but no one outside of the Apple Cult really uses it so Pages isn't a good format to put documents into that you need to submit or collaborate on with others. Now I had been exporting each one to a Word document at the end of each writing session, so that wasn't a big deal, but the readers want one document to review not three!

Reviewer: one or each Word documents?
```suggestion
Up until this point each of my answers were in a separate document in Apple Pages. I prefer Pages because I know where everything is, but no one outside of the Apple Cult really uses it so Pages isn't a good format to put documents into that you need to submit or collaborate on with others. Now I had been exporting each one to Word documents at the end of each writing session, so that wasn't a big deal, but the readers want one document to review not three!
```

I started this week by reviewing everything and making sure that all parts of the questions were answered. Once I was satisfied, each answer was copied into a new Word document only one question wasn't fully answered and stayed where it was until Tuesday. On Tuesday I added the new information I believe I forgot and it was added to the document. I then put the questions in the right order and added the title page the university requires. I was almost done, but still had a bit of work left.

> Now it is Sunday of this fourth week. I have uploaded the document to our plagirism tool to make sure I have properly cited and paraphrised everything. Tomorrow is submission day. I technically have until Tuesday, but, as I mentioned, I don't have much to change or add at the moment so I figure I can submit a day early. Can't hurt anything. Once I submit it though, its just a waiting game.

Reviewer: 1. paraphrased not paraphrised
2. plagiarism not plagirism

```suggestion
Now it is Sunday of this fourth week. I have uploaded the document to our plagiarism tool to make sure I have properly cited and paraphrased everything. Tomorrow is submission day. I technically have until Tuesday, but, as I mentioned, I don't have much to change or add at the moment so I figure I can submit a day early. Can't hurt anything. Once I submit it though, its just a waiting game.
```
